### PR TITLE
Fixes #21090: Create 7.2 release data

### PR DESCRIPTION
**Pull request by peckpeck**
https://issues.rudder.io/issues/21090

### CODE REVIEW

File: release-data/rudder-7.2.cson [hunk 1]
```diff
@@ -0,0 +1,117 @@
+# Release data file
+
+release = "7.1"
```
Comment (amousset) [inline release-data/rudder-7.2.cson:3]:
```suggestion
release = "7.2"
```

File: release-data/rudder-7.2.cson [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+# Status of the release (alpha, beta, rc, final)
+release-status = "final"
```
Comment (amousset) [inline release-data/rudder-7.2.cson:6]:
```suggestion
release-status = "alpha"
```

File: release-data/rudder-7.2.cson [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+
+# 25th April 2022
+release-date = "2022-04-25"
```
Comment (amousset) [inline release-data/rudder-7.2.cson:9]:
```suggestion
#release-date = "2022-04-25"
```

**Comment by peckpeck** [comment]
PR updated with a new commit

**Review by amousset — APPROVED** [review]

**Pull request merged**